src/test/java/core/basesyntax/service/FileWriterTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterImpl;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterTest {
+    private static final String TEST_PATH = "src/main/resources/dayreport.csv";
+    private static final String FILE_HEADER = "fruit,quantity";
+    private static List<String> testList = new ArrayList<>();
+    private static FileWriter fileWriter;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        testList.add(FILE_HEADER);
+        testList.add(System.lineSeparator());
+        testList.add("banana,30");
+        testList.add(System.lineSeparator());
+        testList.add("pineapple,22");
+        testList.add(System.lineSeparator());
+        testList.add("apple,50");
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void writeToFileFileExist_OK() {
+        fileWriter.writeToFile(TEST_PATH, testList);
+        File reportFile = new File(TEST_PATH);
+        assertTrue(reportFile.exists());
+    }
+
+    @Test
+    public void writeToFile_OK() {
+        File file = new File(TEST_PATH);
+        file.delete();
```

Comment:
why do we need this?

Reply:
The program create a report file and a tests create a report file too. Tests add information to file again and, of course, reports become are not equals: in one hand one report and in another hand two reports. So we must clear report file via delete it